python/social_bot/envs/grocery_ground.py

```diff
@@ -85,6 +90,17 @@ def __init__(self,
                 it's considered a failure and is givne reward -1
             random_range (float): the goal's random position range
             random_goal (bool): if ture, teacher will randomly select goal from the object list each episode
+            start_range (float): for curriculum learning, the starting random_range to set the goal
+                Enables curriculum learning if start_range > 1.2 * success_distance_thresh.
```

Comment:
Maybe we can add a bool flag like "use_curriculum_learning" to enable/disable, and make the default to false. This way is more clear.

Reply:
Done